ios/EntitiesConverter.m

```diff
@@ -88,6 +88,7 @@ + (NSDictionary *)convertSKProduct:(SKProduct *)product {
     result[@"isDownloadable"] = @(product.isDownloadable);
     result[@"downloadContentVersion"] = product.downloadContentVersion;
     result[@"downloadContentLengths"] = product.downloadContentLengths;
+    result[@"currencyCode"] = product.priceLocale.currencyCode;
```

Comment:
Our SDK supports iOS from version 9.0. `currencyCode` available from iOS 10.
In this case, iOS 9 will crash. Could you please add the availability check as added below? 🙏

Reply:
OK will do later today